src/lib/pseudo.js

```diff
@@ -1,27 +1,45 @@
 'use strict';
 
-// Recursively walk an AST node searching for content leaves
-export function walkContent(node, fn) {
-  if (typeof node === 'string') {
-    return fn(node);
+// Walk an entry node searching for content leaves
+export function walkEntry(entry, fn) {
```

Comment:
`attrs` should be an object.  I see your point about the `index`.  It can be a string and we shouldn't pseudolocalize it.